src/components/Cluster/NewCluster/ReleaseSelector/ReleaseRow.tsx

```diff
@@ -87,14 +94,17 @@ const ReleaseRow: FC<IReleaseRow> = ({
               onChange={() => selectRelease(version)}
               formFieldProps={{
                 margin: 'none',
+                tabIndex: -1,
```

Comment:
When you navigate the page using your keyboard, you now select the whole release row, as opposed to just the radio input